OfficeAddin/05 Using modern JavaScript/Demos/03 Office Add-in using VueJS/README.md

```diff
@@ -1,7 +1,7 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
 ----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using a VueJS a popular JavaScript framework. 
```

Comment:
Suggest adding a comma after the word "Vue" (i.e., "...using Vue, a popular JavaScript framework.")

Reply:
Fixed.